content/_index.md

```diff
@@ -7,10 +7,28 @@ BookToC: false
 
 # The Linux Userspace API (UAPI) Group
 
-The userspace API ("uapi") group is a community for people with an interest in innovating how we build, deploy, and run modern Linux operating systems.
+The userspace API ("uapi") group is a community for people with an interest in innovating how we build, deploy, run, and safely update modern Linux operating systems.
```

Comment:
The "safely update" makes it read like we only care about the updates to be safe instead of also safely run and deply, and build. If this is an important message for you, then please add a sentence like "with a focus on security` or sm.

Reply:
I meant "securely" in this context - updated.